scripts/install_third_party_test.py

```diff
@@ -143,7 +143,7 @@ def mock_exists(path):
 
         def mock_url_open(unused_url):
             self.check_function_calls['url_open_is_called'] = True
-            # The function is used as follows: python_utils.url_open(req).read()
+            # The function is used as follows: utils.url_open(req).read()
```

Comment:
Is this still true?

Reply:
the function remains unchanged, only moved to `utils.py`. so it still holds true I think.